powerspikegg/computation-models/utils/src/graph.cc

```diff
@@ -0,0 +1,21 @@
+#include "graph.h"
+
+namespace computation {
+namespace utils {
+
+tensorflow::Node* FindNodeWithNameInGraph(tensorflow::Graph* graph,
+                                          std::string name) {
+    for (tensorflow::Node* node : graph->nodes()) {
+        if (node->name() == name) return node;
```

Comment:
We should save a node label map (label -> node) to avoid this worst O(n) case in later PRs.